### DISCUSSION TITLE
Adapt UI language to HA locale

### DISCUSSION THREAD

**Discussion by aftereffexts** [matt8707/hass-config#163] [Ideas]
It is possible to implement a locale function, with dependency which locale is setted up on the Homeassistant Instace ?

For example: a german person wants to implement your config, after implementing the language is written in german instead in swedish.

**Comment by yognspoon** [comment]
Yes, had some fun translating into English.  The words I actually found most difficult were items in US english.  Took me weeks to realise what a mailbox is...  Looking for sensors for my email for days, then the penny dropped.  Mailbox = letter box.  lol.
